openy_af_vue_app/SMOKE_TESTS.md

```diff
@@ -0,0 +1,368 @@
+5.3 On the next step check the Age(s) or skip it (by clicking ""Skip"" link)
+5.4 On the next step you can select or skip days of week
+5.6 On the next step you can select or skip location
+5.6 You should be redirected to result page with filtered activities
+
+
+### Expected Results
+
+1. You should be redirected to results page.
+
+2. Search by keyword:
+2.1 In the left sidebar in the search field you should see keyword from previous step
+2.2. You should see activities that related to entered keyword
+2.3. You should have ability to get more information about activity (by click on activity card)
+2.4. In the modal windows after click by activity card you should have ability to register to this activity
+2.5. You should have ability to filter results byfollowing filters 
+ - Schedule preferences
+ - Programs & Classes
+  Location preferences
+
+3. Filter by age:
+3.1 You can filter activities by Age/ages first
+3.2 On the results page found programs are correct, according to selected filter
+
+4. Filter by week:
+4.1 You can filter activities by Day of week
+4.2 On the results page found programs are correct, according to selected filter
+
+5. Filter by activity:
+5.1 You can filter activities by activities
+5.2 On the results page found programs are correct, according to selected filter
+
+## Check "Start Over"
+
+### User
+
+Anonymous
+
+### Steps
+
+1. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+2. Search by keyword or using one of the available filters 
+3. On any page (except the initial page) you should have a link ""Start over"" in the page header
+4. Click by this link, then you should be redirected on start screen with default states of filters
+
+
+### Expected Results
+
+1. On all pages except the initial page should be displayed a link ""Start Over""
+2. After cicking on the ""Start Over"" link search should start from scratch
+
+## Check filters functionality
+
+### User
+
+Anonymous
+
+### Steps
+
+1. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+2. Click by the ""View all programs"" link
+3. You should be redirected to the results page
+4. Play with filters by selecting different values
+5. Verify results are updating without reloading the page
+6. Verify soring by different options is working
+
+
+### Expected Results
+
+1. In left sidebar you should see filters:
+- Searchbar by searching by kyewords
+- Expandable filters in different groups (see test case below)
+2. By clicking on any filters the results should be updated with counting of selected filter
+3. On the top of results you should be able to sort results: 
+- Sort by Title (A-Z)
+- Sort by Title (Z-A)
+- Sort by Location (A-Z)
+- Sort by Location (Z-A)
+- Sort by Activity (A-Z)
+- Sort by Activity (Z-A)
+4. All manipulations on the page happens without reloading the page
```

Comment:
Let's add a step with checking browser console for errors:

```
5. Open browser console (press F12 on keyboard). Check there are no errors related to AF3 in logs.
```

Reply:
done https://github.com/ymcatwincities/openy_activity_finder/pull/80/commits/65dd9c2829cc8ecb4a6e15bbc2e9edf91bee5059